src/compiler/scala/reflect/macros/compiler/Errors.scala

```diff
@@ -1,7 +1,7 @@
 package scala.reflect.macros
 package compiler
 
-import scala.compat.Platform.EOL
+import java.lang.System.{lineSeparator => EOL}
```

Comment:
There already was `Properties.lineSeparator` too, though it doesn't really add anything at all.